Strategic/Queen Command.cpp

```diff
@@ -2780,6 +2774,7 @@ void EnemyCapturesPlayerSoldier( SOLDIERTYPE *pSoldier )
 	{
 		pSoldier->stats.bLife = 0;
 		pSoldier->iHealableInjury = 0; // added by SANDRO
+		BOOLEAN fMadeCorpse;

```

Comment:
hahaha 100% out of scope, but just look at this in HandleSoldierDeath :rofl:

Reply:
-_-